src/osx/cocoa/dirdlg.mm

```diff
@@ -62,26 +65,40 @@
 
 WX_NSOpenPanel wxDirDialog::OSXCreatePanel() const
 {
-    NSOpenPanel *oPanel = [NSOpenPanel openPanel];
+    NSOpenPanel* oPanel = [NSOpenPanel openPanel];
     [oPanel setCanChooseDirectories:YES];
     [oPanel setResolvesAliases:YES];
     [oPanel setCanChooseFiles:NO];
 
-    wxCFStringRef cf( m_message );
+    wxCFStringRef cf(m_message);
     [oPanel setMessage:cf.AsNSString()];
 
+    if ( !m_title.IsEmpty() )
```

Comment:
```suggestion
    if ( !m_title.empty() )
```
Very minor, but it's preferable to use standard-like methods in the new code.

Reply:
Fixed.

Reply:
(although I always get confused if these functions that don't begin with `Is` will empty the string or just test for empty).